src/utils/dateNormalize.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const dateNormalize = (dateString) => {
+  const dateParts = dateString.split(' ');
+
+  const date = dateParts[0];
+  const time = dateParts[1];
+  const timezone = dateParts[2];
+
+  const normalized = `${date} ${time}+${timezone}`;
+
+  return normalized;
+};
```

Comment:
I can't think of any reason why you need this function, I believe it only hurts the functionality

Reply:
Ohh, I spent hours fixing the problem with date selection, because decided to make spentAt property automatically created by DB, but a problem is that it has its own date format, and js can't recognize it, it throws an error and crashes the server. So the only way I found is this function.

Reply:
sounds like a `kostyl'` then

Reply:
Maybe there is a better way, but I didn't find it.